### PR TITLE
tried better models for the wine prediction

### LINKED ISSUE #8: Better models?
**Using different approaches to get better results**
So far we've considered KNN classifiers and Random Forest Classifiers. It is most likely that a well designed Random Forest would have given the better result of the two. What we're wondering is, do you think there's any better way to solve this problem?
Do you know any other algorithms you'd like to use? A neural network perhaps? Or maybe you'd like to go with a more complex approach and use an ensemble? The choice is yours, blow us away with a well constructed model which gives comparable, if not better results than the initial ones used.

Code in a NEW Jupyter notebook (ipynb) from the beginning and select the feature columns you deem appropriate(with an explanation). You can choose to leave out the visualization part, or can use your own visualizations to draw insights, and can scale the data as you please. We just want you to either come up with an improvement with a better model or use a more complex approach like neural nets to see whether they give us better results. 

Your PR must contain a new ipynb file with comments/explanations where required.

 This is an open ended question with no fixed bounty. You will be awarded based on your final accuracy/innovative approach and other such factors (based on overall quality of notebook as well)

### PR DESCRIPTION
**Pull request by abhishek-mishra17**
Tried to predict the quality of wine with some ensemble techniques and neural networks
1.) support vector classifier :  accuracy of around 73%

2.) Xgboost classifier: accuracy of around 77%

3.) MLP classifier : accuracy of around 75%

4.) Sequential Model: accuracy of around 79%

So the sequential models created outperformed all the other models

Closes #8

### CODE REVIEW

**Comment by pk-95** [comment]
Hey @abhishek-pes , will review this in a bit

**Comment by pk-95** [comment]
Hey @abhishek-pes , I see you've added 2 files, the first one's for issue #7 if I'm not wrong, could you submit that as a separate PR as well for closing the Random Forest issue?

**Comment by abhishek-mishra17** [comment]
oh okay i'll do that

**Comment by abhishek-mishra17** [comment]
@pk-95 actually I have already submitted a pr for issue #7 , so can i just submit it for the #23

**Comment by pk-95** [comment]
Hey @abhishek-pes , yup you've got some good models and I see the approach you've used of grouping the qualities. Great stuff.

**Comment by pk-95** [comment]
!bounty 350

**Comment by techverve** [comment]
!bounty 250
